src/views/Clients/ViewClients/index.jsx

```diff
@@ -30,8 +32,11 @@ import clientsQuery from './clients.graphql';
   },
 }))
 export default class ViewWorker extends PureComponent {
+  componentDidMount() {
+    this.handleClientSearchSubmit();
+  }
```

Comment:
This is going to fetch the list of clients twice. Let's rely on the initial graphql call to perform search the filtered clients. You can achieve that by adding the same logic for `clientsOptions` under `variables`. See other comment.